js/ui/filter_builder/ui.filter_operations_dictionary.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var OPERATION_ICONS = {
+        "=": "equal",
+        "<>": "notequal",
+        "<": "less",
+        "<=": "lessorequal",
+        ">": "greater",
+        ">=": "greaterorequal",
+        "notcontains": "doesnotcontain",
+        "contains": "contains",
+        "startswith": "startswith",
+        "endswith": "endswith"
+    },
+
+    OPERATION_NAME = {
+        "=": "equal",
+        "<>": "notEqual",
+        "<": "lessThan",
+        "<=": "lessThanOrEqual",
+        ">": "greaterThan",
+        ">=": "greaterThanOrEqual",
+        "startswith": "startsWith",
+        "contains": "contains",
+        "notcontains": "notContains",
+        "endswith": "endsWith",
+        "isblank": "isBlank",
```

Comment:
From DevExpress.Data code:
```
AddString(FilteringLocalizerStringId.BlanksName, "(Blanks)");
AddString(FilteringLocalizerStringId.NonBlanksName, "(Non Blanks)");
```

@RomanTsukanov @viterobk "is blank" or "blanks"?

Reply:
Depending on whether you need a singular or plural noun

Reply:
I mean consistency across the platforms

Reply:
English grammar says it should be "is blank" and "is not blank"

Reply:
Ok, you are right.
P.S. "(Blanks)" is the string for the header filter after all